roadmap/implementers-guide/src/node/availability/availability-distribution.md

```diff
@@ -46,12 +46,9 @@ backers, not all of them are required to have the needed chunk. This in turn
 could lead to low throughput, as we have to wait for a fetches to fail,
 before reaching a validator finally having our chunk. We do rank back validators
 not delivering our chunk, but as backers could vary from block to block on a
-perfectly legitimate basis, this is still not ideal. See the following issues
+perfectly legitimate basis, this is still not ideal. See issues [2509](https://github.com/paritytech/polkadot/issues/2509) and [2512](https://github.com/paritytech/polkadot/issues/2512)
 for more information:
```

Comment:
```suggestion
for more information.
```

Reply:
Very confused. I could have sworn I changed that already. Anyway, thanks!